static/js/stream_data.js

```diff
@@ -571,6 +571,14 @@ export function get_stream_privacy_policy(stream_id) {
     return stream_privacy_policy_values.private_with_public_history.code;
 }
 
+export function get_invite_only(stream_name) {
```

Comment:
`is_invite_only_by_stream_name` function on line 582 does the exact same thing as this, infact `is_invite_only_by_stream_name` was formerly named as `get_invite_only` and renamed by Tim on commit [3f4d66109be0265a45713419bbee14394c98414f](https://github.com/zulip/zulip/commit/3f4d66109be0265a45713419bbee14394c98414f). So I think we should use the already available function instead of making a new one.

Reply:
I have removed `get_invite_only` function and used `is_invite_only_by_stream_name`.